components/AnimatedMasoniteLogo/index.js

```diff
@@ -18,8 +18,8 @@ function AnimatedMasoniteLogo({ centered }) {
   }
 
   return (
-    <div className={cx({ centered })}>
-      <Lottie options={defaultOptions} height={400} width={400} />
+    <div className={cx([{ centered, 'black': variant === 'dark' }])}>
```

Comment:
- Centered actually isn't doing anything here. It just returns `true`, rather than the class. Let's remove it. It looks like ModalBackdrop centers for us.
- We need to use a better class name for non-React use.